src/main/java/org/waarp/openr66/dao/database/postgres/PostgreSQLTransferDAO.java

```diff
@@ -0,0 +1,36 @@
+package org.waarp.openr66.dao.database;
```

Comment:
Wrong package (missing postgres) and therefore need `import org.waarp.openr66.dao.database.DBTransferDAO;`